pylint/checkers/utils.py

```diff
@@ -782,15 +783,7 @@ def stringify_error(error):
 
 def decorated_with_property(node: astroid.FunctionDef) -> bool:
     """Detect if the given function node is decorated with a property."""
-    if not node.decorators:
-        return False
-    for decorator in node.decorators.nodes:
-        try:
-            if _is_property_decorator(decorator):
-                return True
-        except astroid.InferenceError:
-            pass
-    return False
+    return _is_property(node)
```

Comment:
Could this be a function of a ``FunctionDef`` instance directly in astroid ?

Reply:
After thinking a bit more about it, I'm not sure anymore using `_is_property` is the right call. If you look at [astroid/bases.py](https://github.com/PyCQA/astroid/blob/1e9a3d18e99010a843e6d0cb06a598ab9ba7a986/astroid/bases.py#L62-L74), there are quite a few names that are assumed to be properties. It might add more false-positives then it removes.

I'll push a second commit to just add `functools.cached_property` to `_is_property_decorator`. That would be the safe choice. Please let me know what you think

Reply:
Sounds good, I never saw this ``POSSIBLE_PROPERTIES `` const in astroid before.